rcl/CMakeLists.txt

```diff
@@ -138,6 +139,7 @@ ament_export_dependencies(rmw_implementation)
 ament_export_dependencies(rosidl_runtime_c)
 ament_export_dependencies(type_description_interfaces)
 
+
```

Comment:
nitpick: minimize vertical whitespace (https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace)

Reply:
also, unrelated changes to whitespace should be avoided in a pr, correct or not